<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>1080</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/newPrefix/Windows (3).png);</string>
   </property>
   <property name="text">
    <string>o</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>451</x>
     <y>191</y>
     <width>250</width>
     <height>349</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;
image: none;
color: red;
font-weight: bold;
font-size: 50px;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>701</x>
     <y>191</y>
     <width>250</width>
     <height>349</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;
image: none;
color: red;
font-weight: bold;
font-size: 50px;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>951</x>
     <y>191</y>
     <width>250</width>
     <height>349</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;
image: none;
color: red;
font-weight: bold;
font-size: 50px;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>1201</x>
     <y>191</y>
     <width>250</width>
     <height>349</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;
image: none;
color: red;
font-weight: bold;
font-size: 50px;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>451</x>
     <y>541</y>
     <width>250</width>
     <height>349</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;
image: none;
color: red;
font-weight: bold;
font-size: 50px;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>701</x>
     <y>541</y>
     <width>250</width>
     <height>349</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;
image: none;
color: red;
font-weight: bold;
font-size: 50px;

</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_7">
   <property name="geometry">
    <rect>
     <x>951</x>
     <y>541</y>
     <width>250</width>
     <height>349</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;
image: none;
color: red;
font-weight: bold;
font-size: 50px;

</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_8">
   <property name="geometry">
    <rect>
     <x>1201</x>
     <y>541</y>
     <width>250</width>
     <height>349</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;
image: none;
color: red;
font-weight: bold;
font-size: 50px;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_9">
   <property name="geometry">
    <rect>
     <x>1510</x>
     <y>10</y>
     <width>401</width>
     <height>191</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">image: none;
border: none;
color: red;
font-weight: bold;
font-size: 100px;</string>
   </property>
   <property name="text">
    <string>Next &gt;&gt;</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>190</y>
     <width>1001</width>
     <height>701</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="1">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_1">
      <property name="styleSheet">
       <string notr="true">image: none;
border: none;
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>80</y>
     <width>661</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 50px;
color: red;
image: none;</string>
   </property>
   <property name="text">
    <string>Select Your Photo </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>910</y>
     <width>1841</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 25px;
color: red;
image: none;</string>
   </property>
   <property name="text">
    <string>Next를 누르고 시간이 소요될 수 있습니다. 화면이 하얘지더라도 기다려주시기 바랍니다.</string>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>gridLayoutWidget</zorder>
  <zorder>label_9</zorder>
  <zorder>pushButton</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>pushButton_3</zorder>
  <zorder>pushButton_4</zorder>
  <zorder>pushButton_5</zorder>
  <zorder>pushButton_6</zorder>
  <zorder>pushButton_7</zorder>
  <zorder>pushButton_8</zorder>
  <zorder>pushButton_9</zorder>
  <zorder>label_10</zorder>
 </widget>
 <resources>
  <include location="../code/backgrounds.qrc"/>
 </resources>
 <connections/>
</ui>
